### PR TITLE
Update Authors

### PR DESCRIPTION
**Pull request by justincormack**
Signed-off-by: Justin Cormack <justin.cormack@docker.com>

hippo reading

### CODE REVIEW

File: .mailmap [hunk 1]
```diff
@@ -22,6 +23,8 @@ David Sheets <david.sheets@docker.com> <sheets@alum.mit.edu>
 Ian Campbell <ian.campbell@docker.com> <ijc25@users.noreply.github.com>
 Ian Campbell <ian.campbell@docker.com> <ijc@docker.com>
 Ian Campbell <ian.campbell@docker.com> <ijc@users.noreply.github.com>
+Isaac Rodman <isaac@eyz.us> <isaac.rodman@healthtrio.com>
+Jeff Wu <JeffWuBJ@users.noreply.github.com>
```
Comment (thaJeztah) [inline .mailmap]: Looks like the sign-off used;

```
Signed-off-by: JeffWuBJ <jeff.wu.junfei@gmail.com>
```

Comment (justincormack) [reply .mailmap]: yes but on his Github page he gives his name as Jeff Wu https://github.com/JeffWuBJ

Comment (thaJeztah) [reply .mailmap]: oh, yes, meant the e-mail address (edited my comment)

Comment (justincormack) [reply .mailmap]: ah yes

**Pull request merged**